plux/build/setuptools.py

```diff
@@ -397,11 +452,12 @@ class DistributionPackageFinder(_PackageFinder):
     correctly if configured.
     """
 
-    def __init__(self, distribution: Distribution):
+    def __init__(self, distribution: Distribution, exclude: t.Optional[t.Iterable[str]] = None):
```

Comment:
could we update the pydoc of _PackageFinder, to add a couple of sentences + example how the exclude works?